solver/src/main.rs

```diff
@@ -148,6 +148,10 @@ struct Arguments {
         parse(try_from_str = shared::arguments::wei_from_gwei)
     )]
     gas_price_cap: f64,
+
+    /// The maximum gas price the solver is willing to pay in a settlement
+    #[structopt(long, env, default_value = "0")]
+    paraswap_slippage_bps: usize,
```

Comment:
Is this more natural as a bps, percentage or float ratio? 1 bps = 0.01% = 0.0001 (unit less).
I personally prefer the ratio but I realize it can be hard to read when there are too many zeroes and in finance maybe bps is more common.

Reply:
I think bps is a quite common unit in finance. The problem with floats is that the conversion to u256 and back may be lossy (but I'm also happy to pass it as a float). Ratios are quite hard to parse from env variable aren't they?

Reply:
`BigDecimal` is also possible as it has a `FromStr` implementation (although might be overkill).